packages/svelte/src/compiler/phases/3-transform/css/index.js

```diff
@@ -106,6 +106,7 @@ const visitors = {
 				const character = state.code.original[index];
 
 				if (regex_css_name_boundary.test(character)) {
+					// TODO FIX
```

Comment:
What's this TODO for?

Reply:
Debug note which I forgot to remove.

I will do it a little later.